src/client/src/app/modules/games/components/add-game/add-game.component.ts

```diff
@@ -44,7 +44,7 @@ export class AddGameComponent implements OnInit {
       duration: 2000
     });
     this.addGame.reset();
-    this.router.navigate(['/games'])
+    this.router.navigate(['/games/users-games-list'])
```

Comment:
I believe this is making a little bug where the next page may load faster then the game being added. so you might have to refresh to see the new game. i would try to make this function async and await the game being added or something. i would have to think for a second about how to implement that sorry lol... but i would still merge this and come back to in because this is a larger PR and it looks good and works well

Reply:
Thanks. I will create a separate PR for this and try to make a research about it...